.github/actions/setup/action.yaml

```diff
@@ -26,3 +26,7 @@ runs:
     - name: Install dependencies
       run: pnpm install --frozen-lockfile --child-concurrency=10
       shell: bash
+
+    - name: Install Playwright Browsers
```

Comment:
Maybe this should just be part of the workflow that playwright runs in? This setup action gets used by pretty much every workflow.